src/plasma/plasma_manager.cc

```diff
@@ -1056,7 +1056,7 @@ void process_fetch_requests(ClientConnection *client_conn,
 }
 
 int wait_timeout_handler(event_loop *loop, timer_id id, void *context) {
-  wait_request *wait_req = context;
+  wait_request *wait_req = (wait_request *) context;
```

Comment:
seems we're still far from being consistent with the new naming convention, but this sounds like a separate PR to me.